datalad/interface/rerun.py

```diff
@@ -360,9 +372,41 @@ def _rerun(dset, results):
                 None, ["git", "cherry-pick", res["commit"]],
                 check_fake_dates=True)
             yield res
+        elif rerun_action == "merge":
+            # We uniquify because content may not change (depending on the
+            # value of --onto).
+            new_parents = unique(
+                filter(None, (new_bases.get(p) for p in res["parents"])))
```

Comment:
Ha, didn't know about None-- would have used `bool` here. Good to know I guess

Reply:
Why do you guess?

Reply:
Because this is not a special behavior which isn't as clear (requires reading the function documentation) as just using `bool` as the comparison/casting function here

Reply:
Agreed.  Will change.